detection/serializers.py

```diff
@@ -25,8 +25,6 @@ class Meta:
             'dt_t_um',
             'nu_area_km2',
             'nu_area_ha',
-            'latitude',
```

Comment:
Did you remove latitude and longitude from detection?

Reply:
Done 745678d